hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -396,9 +397,9 @@ def get_pretty_function_description(f):
     name = f.__name__
     if name == "<lambda>":
         return extract_lambda_source(f)
-    elif isinstance(f, types.MethodType):
+    elif isinstance(f, (types.MethodType, types.BuiltinMethodType)):
         self = f.__self__
-        if not (self is None or inspect.isclass(self)):
+        if not (self is None or self is builtins or inspect.isclass(self)):
```

Comment:
When is `self` builtins? Isn't builtins a module?

Reply:
I'll add a comment... `type(abs) == types.BuiltinMethodType`, and `abs.__self__ is builtins`.  

I was rather surprised, but it looks like this might be a more general implemented-in-C thing so I'll also switch to `inspect.ismodule()`.